src/schema/v2/ArtworkImport/createArtworkImport.ts

```diff
@@ -0,0 +1,84 @@
+import {
```

Comment:
Can we name this file `createArtworkImportMutation`? Then it will be more discoverable in the file tree